.github/workflows/test.yml

```diff
@@ -119,28 +119,16 @@ jobs:
       uses: cachix/install-nix-action@v22
       with:
         install_url: https://github.com/nix-community/nix-unstable-installer/releases/download/nix-2.10.0pre20220822_7c3ab57/install
-        nix_path: nixpkgs=channel:${{ env.CURRENT_STABLE_CHANNEL }}
         extra_nix_config: |
           access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
-    - name: Install ${{ env.CURRENT_STABLE_CHANNEL }} channel
+    - name: Install nix-darwin
       run: |
-        nix-channel --add https://nixos.org/channels/${{ env.CURRENT_STABLE_CHANNEL }} nixpkgs
-        nix-channel --update
-    - name: Install nix-darwin and test result
-      run: |
-        export NIX_PATH=$HOME/.nix-defexpr/channels
-        nix-shell -A installer
-        nix-shell -A installer.check
-    - name: Build simple flake configuration
-      run: |
-        nix build ./modules/examples/flake#darwinConfigurations.simple.system --override-input darwin .
-    - name: Activate derivation of simple flake build
-      run: |
-        ./result/sw/bin/darwin-rebuild switch --flake ./modules/examples/flake#simple --override-input darwin .
-    - name: Rebuild and activate simple flake, but this time using nix-darwins flake interface
+        nix build ./modules/examples/flake#darwinConfigurations.simple.system --override-input darwin . --override-input nixpkgs nixpkgs/${{ env.CURRENT_STABLE_CHANNEL }}
```

Comment:
Wouldn't it make more sense to expose the the darwin tools package directly instead of building an entire dummy system as the main usecase here is bootstrapping an install?

Reply:
Replaced with `nix run .#darwin-rebuild` :+1: